src/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests.csproj

```diff
@@ -41,10 +41,13 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="DictionaryEqualityComparer.cs" />
```

Comment:
This is only used by tests, and the utilities folder contains the tests for the assemblies Utilities components. I didn't want to put it in Mocks since it is not a mock, so just left it at the root.